sw/host/opentitanlib/src/chip/autogen/mod.rs

```diff
@@ -2,7 +2,6 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-// The "english breakfast" variant of the chip doesn't have the same
```

Comment:
That code has been manually edited and now got overwritten my the template rendering.